src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -555,8 +555,8 @@ For more information, visit https://docs.nuget.org/docs/reference/command-line-r
   <data name="Warning_MinVersionNonInclusive" xml:space="preserve">
     <value>{0} does not provide an inclusive lower bound for dependency {1}. An approximate best match of {2} was resolved.</value>
   </data>
-  <data name="Warning_ProjectDependencyMissingLowerBound" xml:space="preserve">
-    <value>Project dependency {0} does not contain an inclusive lower bound. Include a lower bound in the dependency version to ensure consistent restore results.</value>
+  <data name="Warning_ProjectDependencyMissingVersion" xml:space="preserve">
+    <value>Project dependency {0} does not contain a version specified. Include a version in the dependency version to ensure consistent restore results.</value>
```

Comment:
Is "... Include a `version range` in the dependency ..." better?

Reply:
I would word this as:

```
Project dependency {0} does not specify a version.  Include a version for the dependency to ensure consistent restore results.
```

Reply:
I like @jeffkl's suggestion.